test/electron/fixtures/app/preloads/default.js

```diff
@@ -27,5 +30,11 @@ contextBridge.exposeInMainWorld('RunnerAPI', {
   mainProcessLastRunInfo: () => {
     ipcRenderer.send('last-run-info-breadcrumb')
   },
+  renderProcessCrash: () => {
+    process.crash()
+  },
+  childProcessCrash: () => {
+    ipcRenderer.send('child-process-crash')
+  },
```

Comment:
I guess this is leftover from trying to test child process crashes — do we want to keep it?